test/test1.js

```diff
@@ -0,0 +1,265 @@
+      baseUrl: 'http://localhost:3000',
+
+      modules: {
+        'apostrophe-pages': {
+          park: [{slug: '/', type: 'home'}],
+          types: [
+            {
+              name: 'home',
+              label: 'Home'
+            },
+            {
+              name: 'testPage',
+              label: 'Test Page'
+            }
+          ]
+        },
+        'apostrophe-personas': {
+          personas: [
+            {   
+              name: 'employee',
+              label: 'Employee',
+              prefix: '/employee'
+            },  
+            {   
+              name: 'employer',
+              label: 'Employer',
+              prefix: '/employer'
+            }   
+          ]   
+        }
+      },
+      afterInit: function(callback) {
+        assert(apos.modules['apostrophe-personas']);
+        // Should NOT have an alias!
+        assert(!apos.personas);
+        return callback(null);
+      },
+      afterListen: function(err) {
+        assert(!err);
+        done();
+      }
+    });
+  });
+  
+  it('Should load home page', function (done) {
+    return apos.pages.find(apos.tasks.getReq({ locale: 'default-draft' }), { slug: '/' }).toObject(function(err, home) {
+      assert(!err);
+      assert(home.slug === '/');
+      assert(home.type === 'home');
+      done();
+    });
+  });
+  
+  it('Test request for persona via http', function (done) {
+    let j = request.jar()
+
+    return request({url: `${basePath}/?persona=employee`, jar: j}, 'GET')
+    .on('response', res => {
+      assert(true)
+      done()
+    })
+    .on('error', err => {
+      console.log("ERR", err)
+      assert(false)
+      done()
+    })
+  })
+  
+  // test fail on non-get req
+  it('middleware should not touch non-GET reqs', function (done) {
+    const module = apos.modules['apostrophe-personas']
+    const middleware = module.expressMiddleware
+    let req = apos.tasks.getAnonReq()
+    
+    assert(module, 'has personas module')
+    assert(typeof middleware === "function", 'has middleware object')
+    
+    req.method = "POST"
+    req.session = {}
+    
+    
+    middleware(req, req.res, () => {
+      assert(!req.persona, 'does not set persona for POST request')
+      assert(!req.session.persona, 'does not set person to session for POST req')
+      done()
+    })
+  })
+  
+  // test module middleware for GET req
+  it('test persona switcher - redirect via query', function (done) {
+    const module = apos.modules['apostrophe-personas']
+    const middleware = module.expressMiddleware
+    let req = apos.tasks.getAnonReq()
+    
+    assert(module, 'has personas module')
+    assert(typeof middleware === "function", 'has middleware object')
+    
+    req.method = 'GET'
+    req.session = {}
+    req.headers = {
+      'user-agent': userAgents.desktop
+    }
+    req.query = {persona: "employee"}
+    req.data = {} // @@NOTE - personas middleware expects this, not sure where it comes from -pw
```

Comment:
This is standard in Apostrophe, it is made visible to nunjucks templates as the `data` object whenever a page or other template is rendered via the `render` method of a module. If you want stuff to be visible to Nunjucks, you attach it to `data`.